packages/arb-token-bridge-ui/src/components/App/AppContext.tsx

```diff
@@ -49,15 +46,6 @@ function reducer(state: AppContextState, action: Action) {
         }
       }
 
-    case 'layout.set_txhistory_show_sent_tx':
```

Comment:
Removing everything relevant to different tabs for 'Funds sent' and 'Funds received', as we won't need them anymore.